lib/l10n/intl_es.arb

```
@@ -823,6 +823,19 @@
   "add": "Agregar",
 
   "save": "Guardar",
+  "apply": "Aplicar",
+
+  "@_comment_filter_section": "Strings de la sección de filtros",
+  "fiatCurrencies": "Monedas fiat",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)